src/components/nonEmptyListView/nonEmptyList.scss

```diff
@@ -0,0 +1,146 @@
+input[type='search'] {
+  -webkit-appearance: none;
+}
+
+input[type='search'] {
+  -webkit-appearance: textfield;
+}
+
+.search-box {
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  color: $color-font;
+  @include break(48em) {
+    max-width: 18rem;
+  }
+}
+
+h4 {
+  font-family: $font-base;
+  font-weight: 400;
+  line-height: 1.5;
+  letter-spacing: 0.05ch;
+  color: $color-primary;
+
+  &::after {
+    content: '';
+    display: block;
+    height: 0.5em;
+    width: 100%;
+    border-top: 1px dotted $border-dotted-grey;
+  }
+
+  span {
+    background: $pale-blue;
+  }
+}
+
+.list {
+  padding: 0;
+
+  & > * > * {
+    margin-top: 1.8rem;
+  }
+}
+
+.list__item {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 8px 0px 8px 50px;
+  overflow-wrap: anywhere;
+  @include break(37em) {
+    width: 100%;
+  }
+  @include break(64em) {
+    width: 100%;
+  }
+}
+
+.list__item__input {
+  position: absolute;
+  z-index: -1;
+  margin: 0px;
+  padding: 0px;
+  opacity: 0;
+}
+
+.list__item__input:checked + .list__item__label::after {
+  display: block;
+  position: absolute;
+  content: '';
+  height: 3.2ch;
+  width: 3.2ch;
+  left: 7px;
+  top: 7px;
+  background-image: url(../../tick.svg);
+  background-size: contain;
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.list__item__label {
+  flex: 2;
+  margin-left: 10px;
+  text-align: left;
+  cursor: pointer;
+  color: $color-primary;
+  font-family: $font-base;
+  font-weight: 700;
+  text-transform: capitalize;
+
+  &::before {
+    display: inline-block;
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    content: '';
+    margin-right: 10px;
+    height: 4.2ch;
+    width: 4.2ch;
+    border-radius: 50%;
+    border: 2px solid $color-box-shadow;
+  }
+
+  &.soon::before {
+    background-color: $light-red;
+    box-shadow: 2px 2px 0 $color-box-shadow;
```

Comment:
It seems like this border is the same for all labels so maybe it can go in `.list__item__label` instead of each label type?